lib/pymedphys/cli/experimental/pinnacle.py

```diff
@@ -33,7 +34,20 @@ def pinnacle_cli(subparsers):
     return pinnacle_parser
 
 
+def deprecated_export_cli(args):
```

Comment:
Hi @SimonBiggs, this is the solution I have come up with and now shows the appropriate warnings at the right times. Are you happy with this? If so I'll go ahead an merge :)

Reply:
OK yep sure thing. Currently the warning just states: `The Pinnacle Export Module is intended for research purposes only, use with caution.`

I will expand on that with something like:
`The Pinnacle Export Module is intended for research purposes only. Parts of the DICOM conversion are hard-coded and completely ignore your data. This has proven useful for research applications but is not appropriate for clinical use.`